### PR TITLE
Common Followings of Two User

### PR DESCRIPTION
**Pull request by emingure**

### CODE REVIEW

File: KarpuzTwitterApp/handlers/commonfollowings.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import logic
+from handlers.base import BaseHandler, TemplateRendering
+
+
+class CommonFollowingsHandler(BaseHandler, TemplateRendering):
+    """Handler for searching tweets with location and query."""
```
Comment (enisimsar) [inline KarpuzTwitterApp/handlers/commonfollowings.py]: (:

File: KarpuzTwitterApp/logic.py [hunk 2]
```diff
@@ -16,4 +16,55 @@ def get_tweets_with_location_and_query(search_params):
...
 
-    return {'response': True, 'tweets': tweets}
\ No newline at end of file
+    return {'response': True, 'tweets': tweets}
+
+
+def get_followings_of_user(search_params):
+    search_url = '{}1.1/friends/ids.json'.format(TwitterService().get_base_url())
+
+    search_response = get(search_url, headers=TwitterService().get_request_headers(), params=search_params)
```
Comment (enisimsar) [inline KarpuzTwitterApp/logic.py]: variable name?

File: KarpuzTwitterApp/logic.py [hunk 3]
```diff
@@ -16,4 +16,55 @@ def get_tweets_with_location_and_query(search_params):
...
+    # print(search_response.json())
+    followings = set(data['ids'])
+
+    return {'response': True, 'followings': followings}
+
+
+def get_user_details(search_params):
+    search_url = '{}1.1/users/lookup.json'.format(TwitterService().get_base_url())
+
+    search_response = get(search_url, headers=TwitterService().get_request_headers(), params=search_params)
```
Comment (enisimsar) [inline KarpuzTwitterApp/logic.py]: variable name is ambiguous

**Comment by b-onc** [comment]
I get 
`{"error": "Unexpected Error!"}`
because
```
[E 180413 17:06:26 web:1621] Uncaught exception GET /common-followings?user_one=jack&user_two=jeffdean (::1)
    HTTPServerRequest(protocol='http', host='localhost:8080', method='GET', uri='/common-followings?user_one=jack&user_two=jeffdean', version='HTTP/1.1', remote_ip='::1')
    Traceback (most recent call last):
      File "/Users/bahadir/Dropbox/Documents/School/Spring'18/CMPE352/bounswe2018group5/KarpuzTwitterApp/env/lib/python3.6/site-packages/tornado/web.py", line 1541, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/Users/bahadir/Dropbox/Documents/School/Spring'18/CMPE352/bounswe2018group5/KarpuzTwitterApp/handlers/commonfollowings.py", line 24, in get
        'users': users if users['response'] else [],
    TypeError: list indices must be integers or slices, not str
```

You can check my pr for the solution

**Comment by emingure** [comment]
I pushed a commit that fixes bugs. Please review again. @enisimsar @b-onc

**Pull request merged**